zerver/models/streams.py

```diff
@@ -34,6 +34,7 @@ class Stream(models.Model):
     deactivated = models.BooleanField(default=False)
     description = models.CharField(max_length=MAX_DESCRIPTION_LENGTH, default="")
     rendered_description = models.TextField(default="")
+    stream_default_code_block_language = models.TextField(default="")
```

Comment:
should we rename this to `default_code_block_language`?

Reply:
I had been thinking so